assets/src/edit-story/components/mediaPicker/useMediaPicker.js

```diff
@@ -99,6 +97,8 @@ export default function useMediaPicker({
     });
 
     // When an image is selected, run a callback.
+    // The onSelect logic is defined in mediaPane.js/onSelect, which handles the
+    // logic when user click on the button `Insert into page`.
```

Comment:
This is a confusing comment. Readers will wonder -- what does "onSelect" refer to?

A code pointer to mediaPane.js could be useful but it should be moved to the `Uploader.success` callback around line 65. Also use the full file path: `assets/src/edit-story/components/library/panes/media/local/mediaPane.js`.

Reply:
Hmm...I am not sure why `the code pointer to mediaPane.js could be useful but it should be moved to the Uploader.success callback` as it is only related to the function `fileFrame.on('select',  ...)`

Reply:
Got it.